tasks/README.rst

```diff
@@ -65,11 +65,11 @@ Next Steps
 ~~~~~~~~~~
 
 -  Read the `Client Library Documentation`_ for Cloud Tasks API
-   to see other available methods on the client.
+   API to see other available methods on the client.
```

Comment:
API should be removed on the line above (or drop this part of the patch altogether).